user/python-pytest-cov/template.py

```diff
@@ -0,0 +1,29 @@
+pkgname = "python-pytest-cov"
```

Comment:
we removed this before because it's always trivial to patch out (just remove --cov from the pytest args)

Reply:
same goes for the coverage above then transitively